nextflow.config.example

```diff
@@ -206,6 +216,24 @@ profiles {
     }
   }
 
+  // A profile for execution of GEMmaker on a stand-alone machine
+  // using Docker.
```

Comment:
again, these 'localSingularity' and 'localDocker' were added in a previous commit.  Why are they showing up here as new code?

Reply:
I don't know, I think Tyler added them in. But again, if you look at this file on the master branch, these profiles aren't there. So if they were added on the master branch they must have been deleted at some point.

Reply:
Oh, no I think if you remove it on the branch then it will go away for good.  We can leave it as long as there are no differences between what's here and what's on master. I get nervous with git does this.  I don't know if it's really picking up what's on master or pulling it from a different commit that may have changed....  So, I just want to make sure that this section and the section above with the clean_sra code is identical to what's on master.

Reply:
Hmmm... it seems to be missing in the master branch.  But I added that clean_sra

Reply:
Again, it wasn't clean_sra I added but clean_sam.  My brain malfunctioned.